protos/Makefile

```diff
@@ -26,16 +30,31 @@ help:
 .PHONY: clean
 clean:
 	@mkdir -p pb && rm -f pb/pb.pb.go
+	@rm -f $(GOBIN)/protoc-gen-go
+	@rm -f $(GOBIN)/protoc-gen-gofast
 
 .PHONY: check
 check:
-	@./depcheck.sh && \
-		(echo "Upgrading proto libraries before regenerating." ; \
-		go get -u github.com/golang/protobuf/protoc-gen-go ; \
-		go get -u github.com/gogo/protobuf/protoc-gen-gofast)
+	@./depcheck.sh
+
+.PHONY: golang_protobuf
+golang_protobuf:
+	@echo "\033[0;31mRemoving $(GOPATH)/src/github.com/golang/protobuf\033[0m"
```

Comment:
The color codes are unnecessary and do not make much difference to the output. Can avoid it.